yabt/cli.py

```diff
@@ -83,6 +83,7 @@ def make_parser(project_config_file: str) -> configargparse.ArgumentParser:
         PARSER.add('--builders-workspace-dir', default='yabtwork')
         PARSER.add('--default-target-name', default='@default')
         PARSER.add('--force-pull', action='store_true')
+        PARSER.add('-j', '--jobs', type=int, default=1)
```

Comment:
is 1 a good default value?

Reply:
> is 1 a good default value?

it's a safe default value :-) (consistent with any other build tool out there)

Reply:
I think what Avi want is something like: -1 means use everything you have

Reply:
I don't think something like that should be the default value. I think users of build tools expect consistency between different roomse, and the default behavior is single worker.

I can implement support for taking -1 and interpreting it as "use all available cores", but I rather not have it the default.

Reply:
I was just trying to understand - I guess we can get use to running with -j 8 the same way we do for scons